include/revng/Pipes/Helpers.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Model/Binary.h"
+#include "revng/Model/LoadModelPass.h"
+#include "revng/Pipeline/ContainerSet.h"
+#include "revng/Pipeline/Context.h"
+#include "revng/Pipeline/Contract.h"
+#include "revng/Pipeline/GenericLLVMPipe.h"
+#include "revng/Pipeline/LLVMContainer.h"
+#include "revng/Pipeline/Loader.h"
+#include "revng/Pipeline/SavableObject.h"
+#include "revng/Pipeline/Target.h"
+#include "revng/TupleTree/TupleTree.h"
+
+namespace pipeline {
+
+class ModelPipelineWrapper : public SavableObject<ModelPipelineWrapper> {
+private:
+  ModelWrapper Model;
+
+public:
+  constexpr static const char *Name = "RevngWrapper.yaml";
+  static const char ID;
+  llvm::Error storeToDisk(llvm::StringRef Path) const final;
+  llvm::Error loadFromDisk(llvm::StringRef Path) final;
+
+  explicit ModelPipelineWrapper(ModelWrapper Model) : Model(std::move(Model)) {}
+  ModelPipelineWrapper() = default;
+
+  const ModelWrapper &getModelWrapper() const { return Model; }
+
+  ModelWrapper &getModelWrapper() { return Model; }
+};
+
+inline const model::Binary &getModelFromContext(const Context &Ctx) {
+  using Wrapper = ModelPipelineWrapper;
+  const auto &Model = llvm::cantFail(Ctx.getGlobal<Wrapper>(Wrapper::Name));
```

Comment:
nah, it just because otherwise it would be more than 80 characters

Reply:
I mean
```suggestion
  const auto &Model = llvm::cantFail(Ctx.getGlobal<Wrapper>());
```
And `getGlobal` fetches the name from `Wrapper::Name` on its own.

Reply:
this is different than the various register things, since the various user may want to register the same class multiple times, and fetch the same object type multiple times with different name. Here i would force the user to be always explicit about which one it was so that he does not forget to write T::Name and ends up fetching a different one but with the same type.

Reply:
Uhm OK.